### PR TITLE
#3 Issue 3

### PR DESCRIPTION
**Pull request by grantmagdanz**
Some description of the issue.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 subprojects {
-  version = "1.0.0"
+  version = "1.0.1"
```
Comment (grantmagdanz) [inline build.gradle:11]:
```suggestion
  version = "1.0.0"
```

File: tools/src/main/python/dlpx/virtualization/_internal/commands/build.py [hunk 2]
```diff
@@ -11,18 +11,23 @@
...
 
 logger = logging.getLogger(__name__)
 
-TYPE = 'Plugin'
-LOCALE_DEFAULT = 'en-us'
-VIRTUAL_SOURCE_TYPE = 'PluginVirtualSourceDefinition'
-DISCOVERY_DEFINITION_TYPE = 'PluginDiscoveryDefinition'
-STAGED_LINKED_SOURCE_TYPE = 'PluginLinkedStagedSourceDefinition'
-DIRECT_LINKED_SOURCE_TYPE = 'PluginLinkedDirectSourceDefinition'
+TYPE = "Plugin"
```
Comment (grantmagdanz) [inline tools/src/main/python/dlpx/virtualization/_internal/commands/build.py:25]: Formatting looks good

**Comment by muralinimmagadda** [comment]
Reviewed and approving

File: tools/src/main/python/dlpx/virtualization/_internal/click_util.py [hunk 3]
```diff
@@ -57,12 +57,17 @@ def validate_option_exists(ctx, param, value):
         # Let the user know if there is an environment variable for this param
         if param.envvar:
             raise click.BadParameter(
-                ('Option is required and must be specified via '
-                 'the command line or using the environment variable "{}".'.
-                 format(param.envvar)))
+                (
+                    "Option is required and must be specified via "
```
Comment (muralinimmagadda) [inline tools/src/main/python/dlpx/virtualization/_internal/click_util.py:61]: Looks good

File: tools/src/main/python/dlpx/virtualization/_internal/codegen.py [hunk 4]
```diff
@@ -13,17 +13,14 @@
...
 # The Swagger JSON specification requires these following fields.
 SWAGGER_JSON_FORMAT = {
-    'swagger': '2.0',
-    'info': {
-        'version': '1.0.0',
-        'title': None
-    },
-    'paths': {},
-    'definitions': {}
+    "swagger": "2.0",
```
Comment (muralinimmagadda) [inline tools/src/main/python/dlpx/virtualization/_internal/codegen.py:20]: another review comment

**Review by muralinimmagadda — DISMISSED** [review]

File: tools/src/main/python/dlpx/virtualization/_internal/click_util.py [hunk 5]
```diff
@@ -30,9 +30,9 @@ class ConfigFileProcessor(ConfigFileReader):
     The config file processor will search for a config file in the current
     user's home directory.
     """
+
     config_files = [
-        os.path.expanduser(os.path.join('~', CONFIG_DIR_NAME,
-                                        CONFIG_FILE_NAME))
+        os.path.expanduser(os.path.join("~", CONFIG_DIR_NAME, CONFIG_FILE_NAME))
```
Comment (muralinimmagadda) [inline tools/src/main/python/dlpx/virtualization/_internal/click_util.py:35]:
```suggestion
        os.path.expanduser(os.path.join("~", CONFIG_DIR_NAME,
         CONFIG_FILE_NAME))
```

File: tools/src/main/python/dlpx/virtualization/_internal/click_util.py [hunk 6]
```diff
@@ -30,9 +30,10 @@ class ConfigFileProcessor(ConfigFileReader):
     The config file processor will search for a config file in the current
     user's home directory.
     """
+
     config_files = [
-        os.path.expanduser(os.path.join('~', CONFIG_DIR_NAME,
-                                        CONFIG_FILE_NAME))
+        os.path.expanduser(os.path.join("~", CONFIG_DIR_NAME,
+         CONFIG_FILE_NAME))
```
Comment (grantmagdanz) [inline tools/src/main/python/dlpx/virtualization/_internal/click_util.py]:
```suggestion
                                        CONFIG_FILE_NAME))
```

**Review by muralinimmagadda — DISMISSED** [review]
approving

**Comment by muralinimmagadda** [comment]
Looks good

**Review by muralinimmagadda — APPROVED** [review]

**Pull request merged**